packages/modal/src/ModalContainer.tsx

```diff
@@ -35,3 +35,8 @@ ModalContainer.propTypes = {
   restoreFocus: PropTypes.bool,
   environment: PropTypes.any
 };
+
+ModalContainer.defaultProps = {
+  focusOnMount: true,
+  restoreFocus: true
```

Comment:
Would this change in behavior cause unexpected changes for consumers? I think these are sensible defaults... worried about if it may cause issues with those updating.